consensus/Cargo.toml

```diff
@@ -15,13 +15,14 @@ browser = []
 [dependencies]
 mina-crypto = {version = "0.1.0", path = "../crypto"}
 mina-rs-base = {version = "0.1.0", path = "../base"}
-
+snark-keys = {version = "0.1.0", path = "../snark-keys"}
```

Comment:
Where is this being used?

Reply:
See [line](https://github.com/ChainSafe/mina-rs/blob/68acc0c4021d32de6f2f3af11cdc7927b2bfd975/consensus/tests/checkpoint.rs#L65) not sure we will need computes currency at height in the future tests like for staged ledger diff.
I'm reading [staged ledger diff](https://github.com/MinaProtocol/mina/blob/af76cb7980d5e81e704120290a850ea9c6f8522e/src/lib/staged_ledger_diff/staged_ledger_diff.ml) for #113

Reply:
fixed

Reply:
Feels like we should tackle that as a separate PR